plugin.video.filmarkivet/resources/language/resource.language.en_gb/strings.po

```diff
@@ -0,0 +1,54 @@
+# Kodi Media Center language file
```

Comment:
Sorry for so many requests but, for the addon to be in the gotham branch you need to keep the old language layout while moving to gettext:

plugin.video.filmarkivet/resources/language/English/strings.po

Reply:
No problem, it is my fault as I haven't updated the addon very often.
Ok, I will update it to Krypton then, I agree that it should be easier.
If I move to Krypton, then I don't need to keep the old language layout?
I guess I should cancel this PR then and make a new for Krypton?